storage/queue.go

```diff
@@ -198,20 +198,27 @@ func (bq *baseQueue) MaybeRemove(rng *Range) {
 //
 // TODO(spencer): current load should factor into range processing timer.
 func (bq *baseQueue) processLoop(clock *hlc.Clock, stopper *stop.Stopper) {
+
 	stopper.RunWorker(func() {
 		// nextTime is initially nil; we don't start any timers until the queue
 		// becomes non-empty.
 		var nextTime <-chan time.Time
 
+		immediately := make(chan time.Time)
+		close(immediately)
+
 		for {
 			select {
 			// Incoming signal sets the next time to process if there were previously
 			// no ranges in the queue.
 			case <-bq.incoming:
 				if nextTime == nil {
-					// When the first range is added, wake up immediately. This is
-					// mainly to facilitate testing without unnecessary sleeps.
-					nextTime = time.After(0 * time.Millisecond)
+					// When a is added, wake up immediately. This is mainly to
+					// facilitate testing without unnecessary sleeps.
+					nextTime = immediately
```

Comment:
Why make this change? It's a rare occurrence so we're not saving anything by avoiding `time.After(0)` here.